silkworm/core/common/test_util.hpp

```diff
@@ -252,6 +252,20 @@ inline const std::map<std::string, ChainConfig> kNetworkConfig{
          .london_block = 0,
          .terminal_total_difficulty = 0,
      }},
+    {"Paris",  // same as Merge
```

Comment:
Have they changed the name? Is the "Merge" name still relevant?

Reply:
Yes, "Merge" was changed to "Paris" in [v13.1](https://github.com/ethereum/tests/releases/tag/v13.1). I'm keeping "Merge" for backwards compatibility.